book/src/specs/kimchi.md

```diff
@@ -294,6 +294,126 @@ in order to add zero-knowledge to the protocol.
 ### Lookup
 
 
+Because of our ZK-rows, we can't do the trick in the plookup paper of
+wrapping around to enforce consistency between the sorted lookup columns.
+
+Instead, we arrange the LookupSorted table into columns in a snake-shape.
+
+Like so,
```

Comment:
In my computer, this does not render like a snake, but it simply places all | symbols one after the other in the same row... Perhaps you can force newline by \n or leaving one blank space? (Maybe this will give you funny spacing though)

Reply:
```
Or you can have monospace font with block code format
```

Reply:
yeah that'll have to be fixed, getting this PR in for now as it's simply converting comments and fixing CI, but will work on that next